src/util-file.c

```diff
@@ -149,6 +155,25 @@ void FileForceTrackingEnable(void)
     g_file_force_tracking = 1;
 }
 
+void FileWriteMetaDisable(void)
```

Comment:
I don't see how we could use multiple instances of file-store. Doing twice the writing seems a terrible idea.
But do you want me to add a conf to the data field of OutputCtx ? Currently I only see this used by the meta.